tests/ProxyManagerTest/Functional/AccessInterceptorValueHolderFunctionalTest.php

```diff
@@ -698,7 +698,7 @@ public function testWillInterceptAndReturnEarlyOnVoidMethod() : void
                         return;
                     }
 
-                    $instance->counter += 1;
+                    ++$instance->counter;
```

Comment:
Please revert: I want to get rid of `++` and `--` overall in any PHP I work with

Reply:
May you tell me why? Reverted btw

Reply:
It's an additional operator with its own precedence semantics: I'm overall not allowing it at all. Yes, it is only confusing in the context of `if ($a++) {` or `for` constructs, but I'd gladly avoid its overall proliferation.